src/xmipp/libraries/reconstruction/volume_subtraction.cpp

```diff
@@ -110,10 +113,12 @@ class ProgVolumeSubtraction: public XmippProgram
         addParamsLine("[--iter <n=1>]        	: Number of iterations");
         addParamsLine("[--mask1 <mask=\"\">]  	: Mask for volume 1");
         addParamsLine("[--mask2 <mask=\"\">]  	: Mask for volume 2");
+        addParamsLine("[--maskSub <mask=\"\">]  : Mask for subtraction region");
```

Comment:
fix indentation (here of mask, also with the energy flag)

Reply:
sorry again about this, I see correct indentation in my editor ...